ember_debug/route-debug.js

```diff
@@ -168,6 +174,12 @@ function buildSubTree(routeTree, route) {
         // Ember < 3.9.0
         routeHandler = routerLib.getHandler(handler);
       }
+
+      // Skip when route is an unresolved promise
+      if (typeof routeHandler?.then === 'function') {
+        continue;
```

Comment:
@pzuraq @chancancode does this sound like a good fix or should we instead be waiting on the promises here?

Reply:
In my observation, `buildSubtree` keeps getting called until all the promises are settled anyway, so skipping or waiting for the engines in the initial calls go by faster than I can perceive the difference. I'm interested in your thoughts tho.